docs/cli.md

```diff
@@ -377,39 +460,43 @@ done
 | `--start` | ISO-8601 timestamp indicating the start of the interval |`export`, `delete`, `dataset`| Optional |
 | `--end` | ISO-8601 timestamp indicating the end of the interval |`export`, `delete`, `dataset`| Optional |
-| `--file-ids` | File-ids to add or remove from a dataset or add or remove tags from |`dataset`, `tag ids` | **Required** |
-| `--include-jsonl` | Set to `true` to include JSON Lines files for local testing |`dataset export`| Optional |
+| `--file-ids` | File-ids to add or remove tags from | `tag ids` | **Required** |
 | `--location-id` | Location ID for the file ids being added or removed from the specified dataset (only accepts one location id) |`dataset`, `tag ids` | **Required** |
 | `--location-ids` | Filter by specified location ID (accepts comma-separated list). See [Using the `ids` argument](#using-the-ids-argument) for instructions on retrieving these values. |`export`, `delete`, `tag filter`| Optional |
 | `--method` | Filter by specified method |`export`, `delete`| Optional |
 | `--mime-types` | Filter by specified MIME type (accepts comma-separated list) |`export`, `delete`|false |
-| `--org-id` | Org ID for the database user being configured (with `database`), or for the file ids being added or removed from the specified dataset (with `dataset`) | `database configure`, `database hostname`, `dataset`, `tag ids` | **Required** |
+| `--org-id` | Org ID for the database user being configured (with `database`) or data being tagged (`tag ids`) | `database configure`, `database hostname`, `tag ids` | **Required** |
 | `--org-ids` | Filter by specified organizations ID (accepts comma-separated list). See [Using the `ids` argument](#using-the-ids-argument) for instructions on retrieving these values. |`export`, `delete`, `tag filter` | Optional |
-| `--parallel` | Number of download requests to make in parallel, with a default value of 10 for `export` and `delete` and 100 for `dataset export` |`export`, `delete`, `dataset export` | Optional |
+| `--parallel` | Number of download requests to make in parallel, with a default value of 10 |`export`, `delete`, `dataset export` | Optional |
 | `--part-id` | Filter by specified part ID |`export`, `delete`| Optional |
 | `--part-name` | Filter by specified part name |`export`, `delete`| Optional |
 | `--machine-id` | Filter by specified machine ID |`export`, `delete`, `tag filter` | Optional |
 | `--machine-name` | Filter by specified machine name |`export`, `delete`| Optional |
-| `--tags` | Filter by (`export`, `delete`, `dataset`) or add (`tag`) specified tag (accepts comma-separated list) |`export`, `delete`, `dataset`, `tag` | Optional |
+| `--tags` | Filter by (`export`, `delete`) or add (`tag`) specified tag (accepts comma-separated list) |`export`, `delete`, `tag` | Optional |
 | `--password` | Password for the database user being configured | `database configure` | **Required** |
 
 ##### Using the `ids` argument
```

Comment:
why is this still here? This should be with the dataset section

Reply:
Bc its used for data tags as well. Can be used for either